universum/modules/code_report_collector.py

```diff
@@ -39,13 +39,22 @@ def prepare_environment(self, project_configs):
                 continue
 
             self.set_code_report_directory(self.settings.project_root)
-
             temp_filename = "${CODE_REPORT_FILE}"
-            for enum, i in enumerate(item["command"]):
-                if temp_filename in i:
-                    name = utils.calculate_file_absolute_path(self.report_path, item.get("name")) + ".json"
-                    actual_filename = os.path.join(self.report_path, name)
-                    item["command"][enum] = item["command"][enum].replace(temp_filename, actual_filename)
+            name = utils.calculate_file_absolute_path(self.report_path, item.get("name")) + ".json"
+            actual_filename = os.path.join(self.report_path, name)
+
+            for key in item:
+                if key == "command":
+                    for enum, i in enumerate(item[key]):
```

Comment:
Why we need enumerate at all?

Reply:
To address one exact list member when running replace in line 50, of course

Reply:
The i` variable is used only in line 49.
I guess it should work even if you use `item[key][enum]` instead.

Also, I am not sure the `if` on line 49 is needed at all. Can you just always call `replace`?